tests/tests-tax.php

```diff
@@ -131,6 +131,57 @@ public function test_get_global_tax_rate() {
 		$this->assertEquals( '0.036', edd_get_tax_rate() );
 	}
 
+	public function test_get_tax_rate_post() {
+		$_POST['billing_country'] 	= 'US';
+		$_POST['state'] 			= 'AL';
+		$this->assertEquals( '0.15', edd_get_tax_rate() );
+
+		// Reset to origin
+		unset( $_POST['billing_country'] );
+		unset( $_POST['state'] );
+	}
+
+	/**
+	 * @group tesst
```

Comment:
Ahh, actually it should be removed, I use that for testing a single test instead of everything/a file. Thanks! Will remove soon